shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/storage/config/StorageContainerConfiguration.java

```diff
@@ -36,12 +36,12 @@ public interface StorageContainerConfiguration {
      * 
      * @return docker container environments
      */
-    Map<String, String> getEnvs();
+    Map<String, String> getContainerEnvs();
```

Comment:
Please do not use abbreviation in method name

Reply:
done